src/pages/Landing/index.tsx

```diff
@@ -13,30 +18,57 @@ import styled from 'styled-components/macro'
 import { BREAKPOINTS } from 'theme'
 import { Z_INDEX } from 'theme/zIndex'
 
-const PageContainer = styled.div`
+const PageContainer = styled.div<{ isDarkMode: boolean }>`
+  position: absolute;
+  top: 0;
+  padding: ${({ theme }) => theme.navHeight}px 0px 0px 0px;
   width: 100%;
-  position: relative;
   display: flex;
   flex-direction: column;
   align-items: center;
+  scroll-behavior: smooth;
+  overflow-x: hidden;
 
-  height: ${({ theme }) => `calc(100vh - ${theme.navHeight + theme.mobileBottomBarHeight}px)`};
-  @media screen and (min-width: ${({ theme }) => theme.breakpoint.md}px) {
-    height: ${({ theme }) => `calc(100vh - ${theme.navHeight}px)`};
-  }
+  background: ${({ isDarkMode }) =>
+    isDarkMode
+      ? 'linear-gradient(rgba(8, 10, 24, 0) 0%, rgb(8 10 24 / 100%) 45%)'
+      : 'linear-gradient(rgba(255, 255, 255, 0) 0%, rgb(255 255 255 /100%) 45%)'};
 `
 
 const Gradient = styled.div<{ isDarkMode: boolean }>`
   position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   top: 0;
   bottom: 0;
   width: 100%;
+  min-height: 550px;
   background: ${({ isDarkMode }) =>
     isDarkMode
       ? 'linear-gradient(rgba(8, 10, 24, 0) 0%, rgb(8 10 24 / 100%) 45%)'
       : 'linear-gradient(rgba(255, 255, 255, 0) 0%, rgb(255 255 255 /100%) 45%)'};
   z-index: ${Z_INDEX.dropdown};
   pointer-events: none;
+  height: ${({ theme }) => `calc(100vh - ${theme.mobileBottomBarHeight}px)`};
```

Comment:
try flex: 1?

Reply:
this doesn't work because we want the gradient to be the height of the window, not the parent. the parent is taller now since it has content below the fold and is scrollable.